src/services/bunker.py

```diff
@@ -258,21 +258,17 @@ def _calculate_cl_rebase_from(self, cdtp: CommonDataToProcess, prev_blockstamp:
         return cl_rebase
 
     def _calculate_lido_balance_with_vault(
-        self, ref_blockstamp: BlockStamp, lido_validators: List[Validator], vault_balance_at: BlockStamp
+        self, lido_validators: List[LidoValidator], vault_balance_at: BlockStamp
```

Comment:
preferably to use a built-in typing for primitives - `list[LidoValidator]`

Reply:
done